integrations/nlu-snips/src/SnipsNlu.ts

```diff
@@ -0,0 +1,191 @@
+import { join as joinPaths, resolve } from 'path';
+import { v4 as uuidV4 } from 'uuid';
+import {
+  axios,
+  AxiosRequestConfig,
+  AxiosResponse,
+  DynamicEntitiesMode,
+  Extensible,
+  HandleRequest,
+  Jovo,
+  JovoError,
+  JovoRequest,
+  NluData,
+  NluPlugin,
+  OutputTemplate,
+} from '@jovotech/framework';
+import { InputType, Intent, JovoModelData } from 'jovo-model';
+import { SnipsNluConfig, SnipsNluResponse } from './interfaces';
+
+export class SnipsNlu extends NluPlugin<SnipsNluConfig> {
+  install(parent: Extensible): Promise<void> | void {
+    super.install(parent);
+
+    parent.middlewareCollection.use('$output', this.generateDynamicEntities.bind(this));
+  }
+
+  getDefaultConfig(): SnipsNluConfig {
+    return {
+      serverUrl: 'http://localhost:5000/',
+      modelsDirectory: 'models',
+      fallbackLanguage: 'en',
+      serverPath: '',
+      engineId: uuidV4(),
+    };
+  }
+
+  async process(handleRequest: HandleRequest, jovo: Jovo): Promise<NluData | undefined> {
+    const text: string | undefined = jovo.$request.getRawText();
+
+    if (!text) {
+      return;
+    }
+
+    const config: AxiosRequestConfig = {
+      url: '/engine/parse',
```

Comment:
Should this be the `serverPath`?

Reply:
Yes, you're right, will fix it together with the point you made above.